sample_factory/algo/sampling/non_batched_sampling.py

```diff
@@ -86,6 +86,9 @@ def __init__(
         self.last_episode_reward = 0
         self.last_episode_duration = 0
 
+        self.episode_return = 0
```

Comment:
I am not a fan of having to introduce another two variables that semantically do exactly the same as last_episode_reward and last_episode_duration. Can we just keep and reuse the old variables?